### DISCUSSION TITLE
Does Dosbox Staging works on Windows 98?

### DISCUSSION THREAD

**Discussion by MaryooBee** [dosbox-staging/dosbox-staging#3466] [General]
I built a fast Socket 775 computer for Windows 98 gaming and I would like to run DOS games on it using Dosbox Staging. Is there any version that works under Win98?

**Comment by MasterO2** [comment]
@MaryooBee https://dosbox-staging.github.io/about/#non-goals

Additionally, assuming you are running Windows 98 on native hardware, you can just run the games natively within Windows 98, or via something like the [FreeDOS Operating System](https://www.freedos.org/download/).

**Comment by MaryooBee** [comment]
The problem is that we either build a slow computer with ISA slots for music cards to maintain the greatest possible compatibility with DOS, or we build something much faster, e.g. with Core 2 Duo 3GHz+ for more demanding games from the Win98 era, and then we lose compatibility with DOS. For me, the ideal solution would be such a fast computer and DOS emulation from Windows 98. Then I have one computer for everything.

**Comment by FeralChild64** [comment]
No, it does not:
- we are using SDL 2.x library for video output, and it does not support Windows 98
- we are using C++17 (slowly switching to C++20), and AFAIK there are no compilers supporting these C++ versions and targetting Windows 98
- we assume the host OS talks UTF-8
- Windows 98 is 32-bit, and we are slowly dropping 32-bit support
- ...and the list can go on and on

I'm afraid you have to stick to the classic DOSBox. To run games natively, you have to use tools like Mo'Slo / Bremze / CpuSpd (https://www.vogons.org/viewtopic.php?t=74359) to slow down your PC, and SBEMU to emulate classic sound card. JEMM and HIMEMX can help you to limit the amount of memory visible to games.

**Comment by johnnovak** [comment]
DOSBox Staging is about running DOS games on modern hardware and OSes, and taking full advantage of them. That necessarily breaks support with old OSes, e.g. the minimum Windows version we support now is Win 10.
